plugin.rb

```diff
@@ -143,4 +143,85 @@ class Engine < ::Rails::Engine
   add_model_callback(User, :before_destroy) do
     DiscourseReactions::ReactionUser.where(user_id: self.id).delete_all
   end
+
+  module ModifyReportLikes
+    def self.included(base)
+      base.instance_eval do
+        def report_likes(report)
+          report.icon = "emoji-icon"
+          report.modes = [:table]
+
+          report.data = []
+
+          report.labels = [
+            {
+              type: :date,
+              property: :day,
+              title: I18n.t("reports.likes.labels.day")
+            },
+            {
+              type: :number,
+              property: :like_count,
+              title: DiscourseReactions::Reaction.main_reaction_id
+            }
+          ]
+
+          titles = DiscourseReactions::Reaction.where("reaction_users_count IS NOT NULL AND reaction_value!=?", DiscourseReactions::Reaction.main_reaction_id).pluck(:reaction_value).uniq
+
+          titles.each { |title|
+            report.labels << {
+              type: :number,
+              property: "#{title}_count",
+              title: title
```

Comment:
Cant you just do this here:

```
PrettyText.unescape_emoji(CGI::escapeHTML(title))
```

Reply:
But it doesn't change anything. It returns the same string